src/tests/ftest/launch.py

```diff
@@ -3141,7 +3141,9 @@ def _process_core_files(self, test_job_results):
             self._fail_test(self.result.tests[-1], "Process", message, sys.exc_info())
             return 256
 
-        return 0
+        message = "Corefile exist"
```

Comment:
I believe this will always report an error even if no core files are detected as the `core_file_processing.process_core_files(test_job_results, True)` returns 0 if it either successfully processed all the files or there were no files to process.  We should probably update that call to return the number of files processed and any errors.

It would be good to run this PR with more than just the `test_core_files` test tag to ensure this works with tests that do not produce core files - something like `test_always_passes test_core_files`

Reply:
_process_core_file is only called if there are core files
`        if "core files" in summary:
            # Process the core files
            return_code |= self._process_core_files(os.path.split(destination)[0])`

so I thought that if you are in self._process_core_files then there are core files to process?

Reply:
nvm core_files is the core file pattern and not the actual core files so it can exist even if not core files